src/demo/components/common/IntentButton.tsx

```diff
@@ -28,7 +28,8 @@ export function IntentButton(props: IntentButtonProps): React.ReactElement {
             })
             .catch(err => {
                 setButtonState(ButtonStateType.ERROR);
-                alert(`Intent failed with message '${err.message}'`);
+                // Alert commented out as per https://github.com/HadoukenIO/fdc3-service/pull/87/files/d637737b8e74e858d8cc978f1fa9755e3ad6c308#r288216259
+                // alert(`Intent failed with message '${err.message}'`);
```

Comment:
Why is this commented out rather than just deleted? The URL doesn't go anywhere anymore

Reply:
the link was meant to be to [this conversation](https://github.com/HadoukenIO/fdc3-service/pull/87#discussion_r288216259). I left it (and also other commented out `alert` calls) as a reminder that something needs to be done at some point / add a story for error handling